### DISCUSSION TITLE
Change of project name

### DISCUSSION THREAD

**Discussion by shibayan** [polymind-inc/acmebot#741] [Announcements]
There are three types of Acmebot that I am developing for App Service / Container Apps / Key Vault, but in the future I would like to unify them for Key Vault.

Therefore, I would like to change the project name to **Acmebot** simply. I think it will generally be called **Azure Acmebot**.

**Comment by webprofusion-chrisc** [comment]
Is there any potential legal problem with leading the product name using someone else's registered trademark? e.g. should it be something like _Acmebot for Azure_, otherwise it looks like it's an official Azure product which may fall foul of microsoft's own brand protection.

**Reply by webprofusion-chrisc** [reply]
Just Acmebot is good though, a bit like Certbot.
